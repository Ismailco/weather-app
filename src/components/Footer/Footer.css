.footer {
  position: absolute;
  margin-top: 40px;
  padding: 10px;
  width: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
  color: #f3f3f3;
  background-color: rgba(231, 70, 130, 0.7);
  line-height: 2rem;
}

.hearts {
  padding: 5px;
  color: red;
  font-size: 20px;
}

.coffee {
  padding: 5px;
}
