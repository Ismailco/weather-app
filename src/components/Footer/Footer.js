import './Footer.css';
import React from 'react';

function Footer() {
  return (
    <div className="footer">
      <p className="footer-content">
        Created with
        <span className="hearts">&hearts;</span>
        and
        <span className="coffee">&#x2615;</span>
        in Morocco,
        <br />
        This App under GPL-3 License.
      </p>
    </div>
  );
}

export default Footer;
