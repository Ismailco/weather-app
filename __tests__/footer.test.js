/**
 * @jest-environment jsdom
 */
import Footer from '../src/components/Footer/Footer';
import { render } from '@testing-library/react';
import React from 'react';

it('Create the Footer component snapshot', () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment(<Footer />)).toMatchSnapshot();
});
