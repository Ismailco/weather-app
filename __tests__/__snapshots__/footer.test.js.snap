// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create the Footer component snapshot 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <p
      class="footer-content"
    >
      Created with
      <span
        class="hearts"
      >
        ♥
      </span>
      and
      <span
        class="coffee"
      >
        ☕
      </span>
      in Morocco,
      <br />
      This App under GPL-3 License.
    </p>
  </div>
</DocumentFragment>
`;
